pkg/api/http/healthz.go

```diff
@@ -78,9 +78,7 @@ func (a *api) onGetHealthz(w http.ResponseWriter, r *http.Request) {
 
 func (a *api) onGetOutboundHealthz(w http.ResponseWriter, r *http.Request) {
 	if !a.outboundHealthz.IsReady() {
-		msg := messages.ErrOutboundHealthNotReady
-		respondWithError(w, msg)
-		log.Debug(msg)
+		respondWithError(w, messages.ErrOutboundHealthNotReadyService)
```

Comment:
I don't think we should be changing this error code at least in a patch release, as there can be programs which are relying on this code and would be a breaking change.